src/lib/provas/2019/26.json

```
@@ -0,0 +1,18 @@
+{
+    "ano": "2019",
+    "questao": "26",
+    "area": "Fundamentos",
+    "enunciado": " Sobre listas, analise as assertivas abaixo: \n\nI. Objetos podem ser inseridos em uma pilha a qualquer momento, mas apenas o que foi inserido \nmais recentemente (isto é, o último) pode ser removido a qualquer momento. \nII. Em uma fila, os elementos podem ser inseridos a qualquer momento, mas apenas o elemento que \nestá a mais tempo na fila pode ser removido. \nIII. Em uma fila, os elementos são inseridos e removidos de acordo com o princípio “o último que \nentra é o primeiro que sai”. \n\nQuais estão corretas?",
```

Nesse caso, acredito que alguns \n sejam mesmo necessários, no entanto ainda existem alguns que acredito não ser necessarios.
```suggestion
    "enunciado": "Sobre listas, analise as assertivas abaixo: \n\nI. Objetos podem ser inseridos em uma pilha a qualquer momento, mas apenas o que foi inserido mais recentemente (isto é, o último) pode ser removido a qualquer momento. \nII. Em uma fila, os elementos podem ser inseridos a qualquer momento, mas apenas o elemento que está a mais tempo na fila pode ser removido. \nIII. Em uma fila, os elementos são inseridos e removidos de acordo com o princípio “o último que entra é o primeiro que sai”. \n\nQuais estão corretas?",
```